model/shared/src/main/scala/clue/model/GraphQLError.scala

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package clue.model
+
+import cats.Eq
+import cats.syntax.either._
+import cats.syntax.eq._
+import clue.model.GraphQLError.{ Location, PathElement }
+
+final case class GraphQLError(
+  message:    String,
+  path:       List[PathElement],
+  locations:  List[Location],
+  extensions: Map[String, String]
+)
+
+object GraphQLError {
+
+  sealed trait PathElement extends Product with Serializable {
```

Comment:
Added `PathElement` because the spec says that the path can be `String` or `Int`.